src/introduction.md

```diff
@@ -1,46 +1,98 @@
+> ESP8266 series is limited and is not being officially supported by Espressif.
 
-This book is currently a work in progress. A number of sections may be missing information or be missing altogether. If there is a specific topic you would like to see documented please [open an issue].
+## Who This Book is For
 
-If you feel you can contribute something to this book, we encourage you to [create a pull request]!
+This book is intended for people with some experience with Rust, and also assumes rudimentary knowledge of embedded development and electronics. For those without prior experience, we recommend first reading the [Assumptions and Prerequisites] and [Other Resources] sections to get up to speed.
 
-[open an issue]: https://github.com/esp-rs/book/issues/new
-[create a pull request]: https://github.com/esp-rs/book/pulls
+[assumptions and prerequisites]: #assumptions-and-prerequisites
+[other resources]: #other-resources
 
-## Who This Book is For
+### Assumptions and Prerequisites
 
-This book assumes some experience with embedded development and the Rust programming language. Teaching these topics is outside the scope of this book.
+- You are comfortable using the Rust Programming Language, and have written and run applications in a desktop environment.
+- You should be familiar with the idioms of the [2021 edition], as this book targets Rust 2021.
+- You are comfortable developing embedded systems in another language such as C or C++, and are familiar with concepts such as:
+  - Cross-compilation
+  - Common digital interfaces like `UART`, `SPI`, `I²C`, etc.
+  - Memory-mapped peripherals
+  - Interrupts
 
-If you are unfamiliar with either topic, please refer to the resources listed below to help you get started.
+[2021 edition]: https://doc.rust-lang.org/edition-guide/rust-2021/index.html
 
-## Additional Resources
+### Other Resources
 
-Some additional resources can be found below which may prove useful for those less experienced with embedded Rust.
+If you are unfamiliar or less experienced with anything mentioned above, or if you would just like more information about a particular topic mentioned in this book, you may find these resources helpful.
 
 | Resource                        | Description                                                                          |
 | ------------------------------- | ------------------------------------------------------------------------------------ |
 | [The Rust Programming Language] | If you are not familiar with Rust we recommend reading this book first.              |
 | [The Embedded Rust Book]        | Here you can find several other resources provided by Rust's Embedded Working Group. |
-| [The Embedonomicon]             | The nitty gritty details when doing embedded programming in Rust.                    |
+| [The Embedonomicon]             | The nitty-gritty details when doing embedded programming in Rust.                    |
```

Comment:
Not sure if this is the best place, but since we are removing the resources chapter, shall we include the Ferrous System training here? Maybe do a small section with some info?

Reply:
Probably a good idea to include yeah, I'll get that added.